usaspending_api/etl/tests/integration/test_load_multiple_submissions.py

```diff
@@ -263,7 +263,7 @@ def setUp(self):
                 ) (values
                     (1, 1, 1, '1010', 'D', 11111, 111110, -11, -111, null, 'x'),
                     (2, 1, 1, '1010', 'D', 22222, 222220, -22, -222, 'B', 'b'),
-                    (3, 1, 1, '1010', 'D', 33333, 333330, -33, -333, 'L', 'p'),
+                    (3, 1, 1, '1010', 'D', 33333, 333330, -33, -333, 'l', 'p'),
```

Comment:
If you remove the change to `usaspending_api/etl/submission_loader_helpers/disaster_emergency_fund_codes.py` I believe this should still pass.